### ISSUE TITLE
Missing stream button on game

### ISSUE THREAD

**Issue by Lepidos** [ValveSoftware/steam-for-linux#6635]
* Steam client version: 31 October 2019
* Distribution: Debian bullseye on both systems
* Opted into Steam client beta?: No
* Have you checked for system updates?: yes

Issue:
For some games installed on both systems, steam client doesn't displays stream (Iocal). 

Steps for reproducing this issue:
1. Start steam client on both systems
2. Select one game, example a windows/proton game.
3. Game page does not displays stream button

Notes:
This did not occurred when, had old interface steam client on streaming client, and had steam client beta on streaming host. That's why I strongly believe this is a issue caused by the new interface on steam.

**Comment by Lepidos** [comment]
After realizing that I had no space on hard drive and deleting 40GB .xsession-errors file, the stream buttons are back...

**Comment by kisak-valve** [comment]
Hello @Lepidos, thanks for the follow up comment. While it's mildly interesting that Steam misbehaved like this, I don't expect anyone would want their system to be in that state making this a non-issue.

Out of mild curiosity, what log spam filled .xsession-errors?

**Comment by Lepidos** [comment]
Oh no, its happening again... Something must have triggered it again. I remember that when the client desktop had the old steam library interface this didn't happened but every time I opted to the beta(new library interface) some games stream button disappered. Mostly pronton games. And still happens.
I removed the file, trying to open would probably take a long time.

**Comment by Lepidos** [comment]
The no space on disk problem it's not related. On the streaming client machine, in order to find out why there was no space, I changed partitions mounting locations that contained SteamLibrary folders, wich caused steam client to "reset" and having to manually login again, download shader caches again, the good news was that stream buttons came back.

Rebooting the stream host machine, starting steam again only few stream buttons are present.

**Comment by Lepidos** [comment]
Found that changing one game proton version and running the game once makes the stream button reappear on the other machine.

**Comment by Fractal147** [comment]
I'm getting this issue presently - the host is NixOS running steam version 1626824053.
With Steam link on Android - I can stream all games as expected (moreover, can stream BPM directly).
With Steam on MacOS, version 1626824053, I was limited in what games I could see to stream, which do indeed appear to all be titles supported via Proton.

However! @Lepidos's fix - forcing the use of a specific proton version and running the game, does re-enable all titles for visibility. This survives restart of steam on both devices, and it still 'worked' after 'not' selecting any specific proton tool.

**Comment by kisak-valve** [comment]
Hello @Lepidos, @Fractal147, are either of you still experiencing this issue on an up to date system?

**Comment by Lepidos** [comment]
Probably yes. I need to retest.
But isn't much of an issue since I discovered steam-link flatpak app.

**Comment by Fractal147** [comment]
Accepting this behaves oddly when forcing proton version, and the last fix was somewhat 'sticky', I've retested with fresh restarts:
Mac OS (Streaming Client, steam v020, package 1647446817) 
NixOS linux (Streaming Server, steam v020, package 1647446817)
Steam Pay settings on Streaming Server: "Enable steam play for all other titles, Run other titles with Proton 6.3-8)


- Native game, Factorio: Can stream
- Native game: Super Hexagon: Streams, works without issue.
- 'Shortcuts to non steam games': (In this case, proton) Can view, stream, but never becomes interactive.
- - (Client remains on the 'loading' screen with tips and spinning 10-segment loading bar)
- NonNative, proton supported game: Lego the Hobbit. Can stream, but actually never becomes interactive.
- - (Client remains on the 'loading' screen with tips and spinning 10-segment loading bar)
- NonNative proton game: Subnautica: Can stream, black screen on client, but controls from client->server works, can 'play' just can't see.

So in summary, the stream button exists now on everything, hooray. 
There are probably compatibility issues, but at least it tries!

**Comment by kisak-valve** [comment]
Closing as outdated.
